### PR TITLE
doc: add missing parameters description of launch script xml

### PR DESCRIPTION
**Pull request by yuchuyang**
Add allow_trigger_s5, enable_ptm, console_vuart and communication_vuart
descriptions.

Tracked-On: #6138
Signed-off-by: Yang,Yu-chu <yu-chu.yang@intel.com>

### CODE REVIEW

File: doc/tutorials/acrn_configuration_tool.rst [hunk 1]
```diff
@@ -322,6 +322,13 @@ current scenario has:
   Specify whether the User VM power off channel is through the IOC,
   power button, or vUART.
 
+``allow_trigger_s5``:
+  Allow vm to trigger s5 shutdown flow, this flag works with ``poweroff_channel``
```
Comment (dbkinder) [inline doc/tutorials/acrn_configuration_tool.rst:326]:
```suggestion
  Allow VM to trigger s5 shutdown flow, this flag works with ``poweroff_channel``
```

File: doc/tutorials/acrn_configuration_tool.rst [hunk 2]
```diff
@@ -322,6 +322,13 @@ current scenario has:
...
+  'vuart1(pty)' and 'vuart1(tty)' only.
```
Comment (dbkinder) [inline doc/tutorials/acrn_configuration_tool.rst:327]:
```suggestion
  ``vuart1(pty)`` and ``vuart1(tty)`` only.
```

File: doc/tutorials/acrn_configuration_tool.rst [hunk 3]
```diff
@@ -322,6 +322,13 @@ current scenario has:
...
+
+``enable_ptm``:
+  Enable the Precision Timing Measurement(PTM) feature.
```
Comment (dbkinder) [inline doc/tutorials/acrn_configuration_tool.rst:330]:
```suggestion
  Enable the Precision Timing Measurement (PTM) feature.
```

File: doc/tutorials/acrn_configuration_tool.rst [hunk 4]
```diff
@@ -334,6 +341,15 @@ current scenario has:
   configure the shared memory regions for current VM, input format:
   ``hv:/<;shm name>;, <;shm size in MB>;``. Refer to :ref:`ivshmem-hld` for details.
 
+``console_vuart``:
+  Enable a pci based console_vuart. Refer to :ref:`vuart_configuration` for details.
```
Comment (dbkinder) [inline doc/tutorials/acrn_configuration_tool.rst:345]:
```suggestion
  Enable a PCI-based console vUART. Refer to :ref:`vuart_configuration` for details.
```

File: doc/tutorials/acrn_configuration_tool.rst [hunk 5]
```diff
@@ -334,6 +341,15 @@ current scenario has:
...
+
+``communication_vuarts``:
+  List of pci based communication vuart. Refer to :ref:`vuart_configuration` for details.
```
Comment (dbkinder) [inline doc/tutorials/acrn_configuration_tool.rst:348]:
```suggestion
  List of PCI-based communication vUARTs. Refer to :ref:`vuart_configuration` for details.
```

File: doc/tutorials/acrn_configuration_tool.rst [hunk 6]
```diff
@@ -334,6 +341,15 @@ current scenario has:
...
+
+``communication_vuart`` (a child node of ``communication_vuarts``):
+  Enable a pci based communication_vuart with its id. Refer to :ref:`vuart_configuration` for details.
```
Comment (dbkinder) [inline doc/tutorials/acrn_configuration_tool.rst:351]:
```suggestion
  Enable a PCI-based communication vUART with its ID. Refer to :ref:`vuart_configuration` for details.
```

**Review by dbkinder — CHANGES_REQUESTED** [review]

**Comment by wenlingz** [comment]
@yuchuyang make doc fail

New errors/warnings found, please fix them:
==============================================

/home/acrn/workspace/make_doc/pme-yocto-build-make-doc-01/acrn-hypervisor/build/doc/rst/tutorials/acrn_configuration_tool.rst:345: WARNING: undefined label: vuart_configuration (if the link has no caption the label must precede a section header)
/home/acrn/workspace/make_doc/pme-yocto-build-make-doc-01/acrn-hypervisor/build/doc/rst/tutorials/acrn_configuration_tool.rst:348: WARNING: undefined label: vuart_configuration (if the link has no caption the label must precede a section header)
/home/acrn/workspace/make_doc/pme-yocto-build-make-doc-01/acrn-hypervisor/build/doc/rst/tutorials/acrn_configuration_tool.rst:351: WARNING: undefined label: vuart_configuration (if the link has no caption the label must precede a section header)

Makefile:65: recipe for target 'html' failed
make[1]: *** [html] Error 1
make[1]: Leaving directory '/home/acrn/workspace/make_doc/pme-yocto-build-make-doc-01/acrn-hypervisor/doc'
Makefile:129: recipe for target 'doc' failed
make: *** [doc] Error 2
+ error make doc
++ timestamp
++ date +%T
+ echo '07:08:45 ERROR: make doc'
07:08:45 ERROR: make doc

**Comment by yuchuyang** [comment]
@wenlingz  updated the tag of reference.
Have verified locally, it works, thanks.

**Review by dbkinder — APPROVED** [review]
LGTM, thanks!

**Pull request merged**